<launch>
    <param name="/controller_logger_file_name" value = "$(find quad_pos_ctrl)/src/logger/" />
    <param name="/PID_logger_file_name" value = "$(find quad_pos_ctrl)/src/logger/" />
    <node pkg="quad_pos_ctrl" type="quad_pos_ctrl_node" name="adam_pos_ctrl_node" clear_params="true" output="screen">
        <param name="/uav_id" value= "2" />
        <param name="/PID/hover_thrust" value= "0.5"/>
        <param name="/PID/single_state_ctrl_param/P_i/x" value= "0.2"/>
        <param name="/PID/single_state_ctrl_param/P_i/y" value= "0.2"/>
        <param name="/PID/single_state_ctrl_param/P_i/z" value= "0.3"/>

        <param name="/PID/single_state_ctrl_param/P_p/x" value= "1.2"/>
        <param name="/PID/single_state_ctrl_param/P_p/y" value= "1.2"/>
        <param name="/PID/single_state_ctrl_param/P_p/z" value= "1.5"/>

        <param name="/PID/single_state_ctrl_param/V_p/x" value= "1.5"/>
        <param name="/PID/single_state_ctrl_param/V_p/y" value= "1.5"/>
        <param name="/PID/single_state_ctrl_param/V_p/z" value= "1.5"/>

        <param name="/PID/single_state_ctrl_param/V_d/x" value= "0.025"/>
        <param name="/PID/single_state_ctrl_param/V_d/y" value= "0.025"/>
        <param name="/PID/single_state_ctrl_param/V_d/z" value= "0.025"/>

        <param name="/PID/all_state_ctrl_param/P_i/x" value= "0.05"/>
        <param name="/PID/all_state_ctrl_param/P_i/y" value= "0.05"/>
        <param name="/PID/all_state_ctrl_param/P_i/z" value= "0.06"/>

        <param name="/PID/all_state_ctrl_param/P_p/x" value= "1.2"/>
        <param name="/PID/all_state_ctrl_param/P_p/y" value= "1.2"/>
        <param name="/PID/all_state_ctrl_param/P_p/z" value= "1.2"/>

        <param name="/PID/all_state_ctrl_param/V_p/x" value= "2.4"/>
        <param name="/PID/all_state_ctrl_param/V_p/y" value= "2.4"/>
        <param name="/PID/all_state_ctrl_param/V_p/z" value= "2.7"/>

        <param name="/PID/all_state_ctrl_param/V_d/x" value= "0.025"/>
        <param name="/PID/all_state_ctrl_param/V_d/y" value= "0.025"/>
        <param name="/PID/all_state_ctrl_param/V_d/z" value= "0.025"/>

        <param name="/PID/ctrl_limit/vel_xy_limit" value= "2.01"/>
        <param name="/PID/ctrl_limit/vel_z_limit" value= "2.01"/>
        <param name="/PID/ctrl_limit/acc_xy_limit" value= "5.01"/>
        <param name="/PID/ctrl_limit/acc_z_limit" value= "5.01"/>
    </node>
</launch>